gralog-core/src/test/java/gralog/dialog/DialogTest.java

```diff
@@ -54,6 +54,12 @@ public void initLists(){
         parameters.clear();
     }
 
+    /*
+    TODO: The following tests are broken because they don't work
+    without user interaction (they try to open dialogs).  For
+    continuous integration (Travis), the tests need to run fully
+    automatic.
+
```

Comment:
It's called dialog because it tests the dialog class, but the tests don't open any dialogue, everything goes fully automatically.

Reply:
Right, I must have been confused, sorry about that.

Checking again, there seems to be only one test that opens a dialog:
`gralog-fx/src/test/java/gralog/gralogfx/windows/PreferenceWindowTest.java`
For this test I'm very sure of it because all it does it calling `Application.launch()`.